scripts/ci/install-istio.sh

```diff
@@ -3,17 +3,17 @@
 DIR=$(dirname $0)
 
 OS=linux
-TARGET_DIR=/usr/bin
+TARGET_DIR=/opt
```

Comment:
will not work as you need a path which exists both for `root` and `vagrant` users

Reply:
Right now, `kubectl` is called for `TestIstioDestinationRuleNode`...

Reply:
yes that is fine